### ISSUE TITLE
Announcing ofjustpy: A webdev framework build/derived from justpy  (link: https://github.com/ofjustpy/ofjustpy)

### ISSUE THREAD

**Issue by sandeep-gh** [justpy-org/justpy#704]
### Discussed in https://github.com/justpy-org/justpy/discussions/703

<div type='discussions-op-text'>

<sup>Originally posted by **sandeep-gh** December  5, 2023</sup>
After using justpy for about couple of months, I had started working some extension and revisions to the framework on my own. 
The result of which is the ofjustpy framework which you can check it out at (https://github.com/ofjustpy/ofjustpy). There are many enhancements/modifications to justpy framework. Three major ones being: 
- Use of svelte as the client side rendering engine 
- Many programmatic optimizations 
- Optimized communication based on difference/changes in the webapp state. 

Do checkout the examples directory in the repo and also this webpage (https://ofjustpy.webworks.monallabs.in/)  build entirely in ofjustpy. 

# Whats lacking
- Currently external javascript/svelte libraries are not incorporated. It easily doable, once we know which is good svelte library to incorporate into the framework.

- Documentation is bit lacking. Justpy users may find using the framework a bit easier. See the README, checkout the examples (https://github.com/ofjustpy/ofjustpy/tree/main/examples), and [htmlcomponents.py](https://github.com/ofjustpy/ofjustpy/blob/main/src/ofjustpy/htmlcomponents.py) for list of htmlcomponents supported. 

If you do checkout the library -- your feedback would be quite valuable. Thank you. </div>

**Comment by longnetwork** [comment]
I started with nicegui and switched to justpy. I needed only quasar. I turned off everything and left only quasar. I understand justpy's architecture better. I don't want nicegui - it's too buggy, too slow, not suitable for high-load projects. It's too labor-intensive to create new components in nicegui - it's buggy, doesn't work, hard to fix. I want justpi - as intended and to develop further!
p.s.) I even turned off tailwind - if you use quasar, then you don't need anything else... maybe if justpy focuses only on one frontend, it will be a great alternative to nicegui, although for me it's already the best alternative.
